server/dataone_package.py

```diff
@@ -93,63 +114,179 @@ def create_minimum_eml(tale, user, item_ids, eml_pid):
     ET.SubElement(ind_name, 'givenName').text = firstName
     ET.SubElement(ind_name, 'surName').text = lastName
 
-    # Add a <otherEntity> block for each object
-    for item_id in item_ids:
-        item = Item().load(item_id, user=user)
-        description = item['description']
-
+    def create_other_entity(name, description):
+        """
+        Create an otherEntity section
+        :param name: The name of the object
+        :param description: The description of the object
+        :type name: str
+        :type description: str
+        :return:
+        """
         other_entity = ET.SubElement(dataset, 'otherEntity')
-        ET.SubElement(other_entity, 'entityName').text = item['name']
-        ET.SubElement(other_entity, 'entityDescription').text = description if\
+        ET.SubElement(other_entity, 'entityName').text = name
+        ET.SubElement(other_entity, 'entityDescription').text = description if \
```

Comment:
entityDescription is optional, so you could just leave it out if it's not present in the backend